packages/sanes-browser-extension/src/routes/wallet/components/SidePanel/PanelDrawer/components/Requests/ShowRequest/index.tsx

```diff
@@ -26,6 +26,7 @@ const ShowRequest = ({ goBackToList }: Props): JSX.Element => {
 
   const closeExtension = (): void => {
     if (requestContext.requests.length === 0) {
+      goBackToList();
```

Comment:
You can move it out of the if condition. Also please leave comment about this behaviour.